share/spice/bike_sharing/indego_phl/content.handlebars

```diff
@@ -0,0 +1,15 @@
+<div class="tile__body">
+    <h1 class="zci__header tx--l4">{{title}}</h1>
```

Comment:
haha, this was tough to figure out! I noticed the font sizes were wrong. Turns out you have `tx--l4` not `tx--14`. Please change these so it has the number `14`, not `l4` and then your font sizes will match the CitiBike Spice.